client/src/AppBundle/Controller/Admin/Client/ClientController.php

```diff
@@ -27,6 +29,53 @@ public function detailsAction($id)
     {
         $client = $this->getRestClient()->get('v2/client/' . $id, 'Client');
 
+        return [
+            'client'      => $client,
+            'namedDeputy' => $this->getNamedDeputy($id, $client)
+        ];
+    }
+
+    /**
+     * @Route("/{id}/discharge", name="admin_client_discharge", requirements={"id":"\d+"})
+     * @Security("has_role('ROLE_SUPER_ADMIN')")
+     * @param Request $request
+     * @param $id
+     *
+     * @Template("AppBundle:Admin/Client/Client:discharge.html.twig")
+     *
+     * @return array
+     */
+    public function dischargeAction(Request $request, $id)
```

Comment:
Don't think request is required here
```suggestion
    public function dischargeAction($id)
```

Reply:
Done 👍